lib/spack/spack/build_environment.py

```diff
@@ -109,7 +109,7 @@ def __call__(self, *args, **kwargs):
         disable = env_flag(SPACK_NO_PARALLEL_MAKE)
         parallel = not disable and kwargs.get('parallel', self.jobs > 1)
 
-        if parallel:
+        if parallel and 'bazel' not in self.command:
```

Comment:
I think the `'bazel' not in self.command` logic can be removed since Bazel actually can build in parallel. Meaning that both subclasses would be identical (I think Bazel also uses `-j<num>` but correct me if I'm wrong).

@muffgaga If you still want to build in serial, you can run `spack install -j1 <package>`. If a package _has_ to be built in serial, you can add `parallel=False` to that `package.py`.